miden-lib/asm/eoa/basic.masm

```diff
@@ -1,32 +1,47 @@
 use.miden::sat::account
 use.miden::sat::tx
+use.std::crypto::dsa::rpo_falcon512
 
-#! Authenticate a transaction using Falcon
-#! WIP - FALCON VERIFICATION IS MISSING, NOT SURE IF HASHING IN CORRECT ORDER
+# CONSTANTS
+# =================================================================================================
+
+# Slot in account storage at which the public key is stored.
+const.PUBLIC_KEY_SLOT=0
+
+#! Authenticate a transaction using Falcon 
```

Comment:
minor nit: "Authenticate a transaction using Falcon" -> "Authenticate a transaction using the Falcon signature scheme"